docs/language/built-in-functions.md

```diff
@@ -31,3 +31,17 @@ title: Built-in Functions
 
   Follow [best practices](https://github.com/ConsenSys/smart-contract-best-practices/blob/051ec2e42a66f4641d5216063430f177f018826e/docs/recommendations.md#remember-that-on-chain-data-is-public)
   to prevent security issues when using this function.
+
+## rlpDecodeString
```

Comment:
We want to make namespace the RLP functionality by exposing it as functions in a contract, e.g. `RLP.decodeString` here. We can do that in a follow-up PR though and can leave the current implementation as-is, to avoid making this PR bigger.

At least we should keep the naming consistent with existing global functionality like BLS (e.g. `AggregateBLSSignatures`), i.e. rename this function to `DecodeRLPString` for now.

(See discussion https://axiomzen.slack.com/archives/CG0B7CJAJ/p1644270003698399)

Reply:
renamed them for now until naming is finalized.